code/modules/mob/living/carbon/human/death.dm

```diff
@@ -101,13 +101,12 @@
 	SetJitter(0)
 	heart_attack = 0
 
+	for(var/obj/item/organ/O in organs) //these are external organs only: arms, legs, etc, internal organs are handled by carbon/death()

```

Comment:
Ugh, there's inconsistencies in the code...there's some instances where internal organs are stored in `organs`.

This is what happens when you name a damn var a poor name; it confuses people (not your fault, of course!).

I'm thinking and wondering if this proc should only apply to internal organs, now that I really think about it; it's only internal organs that have the `on_life` property. It may be best to utilize a different proc name for external, considering there's no mirrored functionality (and poor list naming conventions).

Yeah, let's go with that; just push it up one to `var/obj/item/organ/internal`